user-manual/ingest/manual-normalization.rst

```diff
@@ -4,261 +4,357 @@
+This workflow assumes that users have created access and/or preservation copies
+of their files before the material is transferred into Archivematica.
 
-                /access
+For example, during a digitization project, high-resolution original (master)
+images as well as lower-resolution access copies may be created. It would be
+redundant of Archivematica to duplicate this normalization work. By adding a
+folder called ``manualNormalization`` to the transfer prior to processing, the
+Archivematica can recognize that normalization work has already been done so
+that Archivematica does not replicate the effort.
 
-                    file1.jpg
+.. note::
+   This workflow requires a one-to-one relationship an original object and its
+   access or preservation derivative. Creating multiple access or preservation
+   derivatives that relate to a single original object is not supported.
+
+#. Create your files - the original (master) copies and the manually normalized
+   derivatives. Note that the filenames must match in order for Archivematica to
+   recognize the links between the original file and its derivatives.
+
+   .. code:: bash
+
+      to-be-preserved/
+      ├── hooded-pitcher-plant.gif
+      ├── hooded-pitcher-plant.png
+      ├── hooded-pitcher-plant.tif
+      ├── ornamental-onion.gif
+      ├── ornamental-onion.png
+      ├── ornamental-onion.tif
+      ├── pepperbox-poppy.gif
+      ├── pepperbox-poppy.png
+      └── pepperbox-poppy.tif
+
+   In this example, there are three copies of each file - a GIF, a PNG, and a
+   TIFF.
+
+#. Create a transfer directory and place the original files inside at the top
+   level.
+
+   .. code:: bash
+
+      my-manually-normalized-material/
+      ├── hooded-pitcher-plant.png
+      ├── ornamental-onion.png
+      └── pepperbox-poppy.png
+
+   In the directory structure above, the original objects are PNG files.
+
+#. Create a subdirectory called ``manualNormalization`` (note the capital N).
+   Inside of the ``manualNormalization`` directory, create directories called
+   ``access`` and ``preservation`` (you do not need both - if you only have
+   preservation copies, create only the ``preservation`` directory, and vice
+   versa). Place your manually normalized files in the appropriate directory.
+
+   .. code:: bash
+
+      my-manually-normalized-material/
+      ├── manualNormalization
+      │   └── access
+      │       ├── hooded-pitcher-plant.gif
+      │       ├── ornamental-onion.gif
+      │       └── pepperbox-poppy.gif
+      │   └── preservation
+      │       ├── hooded-pitcher-plant.tif
+      │       ├── ornamental-onion.tif
+      │       └── pepperbox-poppy.tif
+      ├── hooded-pitcher-plant.png
+      ├── ornamental-onion.png
+      └── pepperbox-poppy.png
+
+   In the example above, each original PNG file has a manually normalized access
+   derivative (in GIF format) and a manually normalized preservation derivative
+   (in TIFF format).
+
```

Comment:
I guess if you use my suggestion above this statement would no longer be necessary! Either way works I just like to know where things are going as I read :)